WGS_pipeline/gatk_mutect2_v21.wdl

```diff
@@ -1130,4 +1152,38 @@ task Funcotate {
          File funcotated_output_file = "~{output_file}"
          File funcotated_output_file_index = "~{output_file_index}"
      }
+}
+
+
+task RemoveFiltered {
```

Comment:
we also have a separate wdl that does this. is it redundant here? given it defaults do not running this task

Reply:
nvm! it's removed in mutect2_v4.2.6.1.wdl, which is the one we're currently using